Packs/Cyberint/Integrations/Cyberint/Cyberint.py

```diff
@@ -504,20 +540,53 @@ def fetch_incidents(client: Client, last_run: Dict[str, int],
                         attachments.append(tmp_attachment)
 
         alert["attachments"] = attachments
-        alert_csv_id = alert.get('alert_data', {}).get('csv', {}).get('id', '')
-        if alert_csv_id:
-            extracted_csv_data = extract_data_from_csv_stream(client, alert_id,  # type: ignore
-                                                              alert_csv_id)
-            alert['alert_data']['csv'] = extracted_csv_data
+        incident = {}
+
+        incident_csvs = dict_safe_get(alert, ['alert_data', 'csv', 'content']) or []
+
+        if not fetch_assign_multiple:
+            if incident_csvs:
+                index = 1
+                for incident_csv in incident_csvs:
+                    alert.update({'ref_id': f'{alert_id} ({index})'})
+                    alert['alert_data']['csv'].update({'content': incident_csv})
+
+                    incident = {
+                        'name': f'Cyberint alert {alert_id} ({index}): {alert_title}',
+                        'occurred': datetime.strftime(alert_created_time, DATE_FORMAT),
+                        'rawJSON': json.dumps(alert),
+                        'severity': SEVERITIES.get(alert.get('severity', 'low'), 1),
+                        'attachment': incident_attachments,
+                    }

```

Comment:
This logic is duplicated so let's consider moving this logic out to a helper function.
In addition the constants 1, 'low are representing a magic number of severity. Let's define them as constants.

Reply:
OK